appveyor.yml

```diff
@@ -14,21 +17,39 @@ environment:
     - TARGET_ARCH: "x86"
       CONDA_PY: "27"
       PY_CONDITION: "python >=2.7,<3"
+      MSYS2_ARCH: i686
+      MSYS2_BITS: 32
+      MSYSTEM: MINGW32
```

Comment:
This might be totally unnecessary if we can install these devtools as conda packages, which sounds totally feasible now.